### ISSUE TITLE
IndexOutOfRangeException after importing XRTK.Oculus module via UPM

### ISSUE THREAD

**Issue by FejZa** [XRTK/Oculus#90]
# XRTK - Mixed Reality Toolkit Bug Report

## Describe the bug

The following exception occurs at step 3 of the Testplan provided below when importing the Oculus module via UPM:
```
IndexOutOfRangeException: Index was outside the bounds of the array.
XRTK.Editor.PackageInstaller.AddConfigurations (System.Collections.Generic.List`1[T] profiles) (at Library/PackageCache/com.xrtk.core@0.2.0-preview.113/Editor/PackageInstaller.cs:142)
XRTK.Editor.PackageInstaller+<>c__DisplayClass1_0.<TryInstallAssets>b__0 () (at Library/PackageCache/com.xrtk.core@0.2.0-preview.113/Editor/PackageInstaller.cs:125)
UnityEditor.EditorApplication.Internal_CallDelayFunctions () (at C:/buildslave/unity/build/Editor/Mono/EditorApplication.cs:209)
```

## How to reproduce

Follow Testplan https://github.com/XRTK/XRTK-Core/blob/development/articles/testplans/TP-01_Import-XRTK.Oculus-Without-XRTK.SDK.md

the error will happen at step 3.

**Comment by StephenHodgson** [comment]
I wasn't able to repro this with the latest preview packages using 2019.4.15 LTS.

**Comment by StephenHodgson** [comment]
@FejZa can you confirm this?

**Comment by FejZa** [comment]
I can confirm this seems resolved with 2019.4.15 LTS. I wasn't able to reproduce anymore.

**Comment by FejZa** [comment]
As we are close to updating to the LTS version I think we don't need to look into this further.
